doc/sphinx/running.rst

```diff
@@ -308,11 +343,11 @@ The following minimal example illustrates how to use the SDM in |es|::
 
     import espressomd
     system = espressomd.System(box_l=[1.0, 1.0, 1.0])
+    system.periodicity = [False, False, False]
```

Comment:
why?

Reply:
the SD method doesn't support periodic boundary conditions